### ISSUE TITLE
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = -1.000

### ISSUE THREAD

**Issue by ayennam** [google/automl#1105]
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
INFO:tensorflow:Inference Time : 23.10228s
I1022 18:26:55.658624 140307941200064 evaluation.py:273] Inference Time : 23.10228s
INFO:tensorflow:Finished evaluation at 2021-10-22-18:26:55
I1022 18:26:55.658975 140307941200064 evaluation.py:276] Finished evaluation at 2021-10-22-18:26:55
INFO:tensorflow:Saving dict for global step 893: AP = -1.0, AP50 = -1.0, AP75 = -1.0, AP_/airplane = 0.0, AP_/backpack = 0.0, AP_/bird = 0.0, AP_/bus = 0.0, AP_/car = 0.0, AP_/cat = 0.0, AP_/cow = 0.0, AP_/dog = 0.0, AP_/handbag = 0.0, AP_/horse = 0.0, AP_/motorcycle = 0.0, AP_/person = 0.0, AP_/sheep = 0.0, AP_/toothbrush = 0.0, AP_/train = 0.0, AP_/truck = 0.0, AP_/zebra = 0.0, APl = -1.0, APm = -1.0, APs = -1.0, ARl = -1.0, ARm = -1.0, ARmax1 = -1.0, ARmax10 = -1.0, ARmax100 = -1.0, ARs = -1.0, box_loss = 0.0, cls_loss = 332.13992, global_step = 893, loss = 332.23172
I1022 18:26:55.659120 140307941200064 estimator.py:2074] Saving dict for global step 893: AP = -1.0, AP50 = -1.0, AP75 = -1.0, AP_/airplane = 0.0, AP_/backpack = 0.0, AP_/bird = 0.0, AP_/bus = 0.0, AP_/car = 0.0, AP_/cat = 0.0, AP_/cow = 0.0, AP_/dog = 0.0, AP_/handbag = 0.0, AP_/horse = 0.0, AP_/motorcycle = 0.0, AP_/person = 0.0, AP_/sheep = 0.0, AP_/toothbrush = 0.0, AP_/train = 0.0, AP_/truck = 0.0, AP_/zebra = 0.0, APl = -1.0, APm = -1.0, APs = -1.0, ARl = -1.0, ARm = -1.0, ARmax1 = -1.0, ARmax10 = -1.0, ARmax100 = -1.0, ARs = -1.0, box_loss = 0.0, cls_loss = 332.13992, global_step = 893, loss = 332.23172

**Comment by BhandarkarPawan** [comment]
@ayennam I have the same issue. Were you able to solve it?

**Comment by suke138** [comment]
i face same issue, when i rebuild the workspace again, the issue resolved
